### ISSUE TITLE
run after install fails with

### ISSUE THREAD

**Issue by mschultheiss83** [luminati-io/luminati-proxy#8]
after successfully installing with  
```
npm install -g luminati-io/luminati-proxy
```

try to start with:
```
luminati --customer email/*REGISTRED_EMAIL* --password *SECRED_PASSWORD*
```

getting the following error:
```
/usr/lib/node_modules/luminati-proxy/bin/luminati.js:4
const Manager = require('../lib/manager.js');
^^^^^
SyntaxError: Use of const in strict mode.
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:945:3
```

**Comment by mschultheiss83** [comment]
found my bug: wrong node or npm version
please add to package.json
```
"engines":{"node":version},
```

**Comment by lee-elenbaas** [comment]
Will be fixed in one of the next releases - thanks for pointing it to us

**Comment by lee-elenbaas** [comment]
fixed on v0.5.5
